tests/openfl/utilities/test_fed_timer.py

```diff
@@ -0,0 +1,139 @@
+# Copyright (C) 2020-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Federation Components Timeout tests module."""
+
+import asyncio
+import os
+import pytest
+import time
+
+from openfl.utilities.fed_timer import fedtiming
+from unittest import mock
+
+
+@pytest.mark.parametrize(
+    'input,expected', [
+        ('first', 'first'),
```

Comment:
Looks like one pair is enough. Each test include timeouts that increased time execution significantly. If you want to test that testing decorator doesn't change returning values, probably, it is better to do it separately, not in each test type and only for edge cases.

Reply:
Sure, I thought the timeout decorator must work on any given data type. 
Will limit to just one pair in this case and also reduce the timeout period.
I tried to simulate the part where the execution completes within the time limit and exceeds the time limit.
Will choose a smaller timeout value (Y)